big-practice/src/js/templates/movie.template.js

```diff
@@ -0,0 +1,19 @@
+export class MovieTemplate {
+  static renderTable(move) {
```

Comment:
Should be renderTableRow? Btw what does `test` mean ??

Reply:
Updated!